semgrep_output_v1.atd

```diff
@@ -1528,6 +1528,11 @@ type apply_fixes_params
   edits: edit list;
 }
 
+(* The first int is the number of modified files.
```

Comment:
I think a record would be better than this complex tuple. Then you could use
nice field names in the OCaml and Python code.